src/applications/personalization/profile-2/components/ProfileWrapper.jsx

```diff
@@ -92,7 +93,7 @@ class ProfileWrapper extends Component {
         {/* Breadcrumbs */}
         <Breadcrumbs className="vads-u-padding-x--1 vads-u-padding-y--1p5 medium-screen:vads-u-padding-y--0">
           <a href="/">Home</a>
-          {!onPersonalInformationMobile && <a href="/profile-2/">Profile</a>}
+          {!onPersonalInformationMobile && <Link to="/">Profile</Link>}
```

Comment:
Is `onPersonalInformationMobile` necessary? I thought the `Breadcrumbs` component should handle hiding that but I'm not sure.

Reply:
Yeah, it is so that we are not showing the Profile link - but only on the Personal information page :)

Reply:
Sorry I'm totally late to this! But would it be possible to use `activeRouteName` or `activeLocation` on line 88 instead of digging into the `location.pathname`? And we should probably be comparing to the the value here instead of a string https://github.com/department-of-veterans-affairs/vets-website/blob/69191d71bd4a4f703dfa49d4a27a77a507635419/src/applications/personalization/profile-2/routes.jsx#L9 Not a huge deal, but something we might want to change next time we mess with this file